ledfx/devices/lifx.py

```
@@ -17,6 +17,7 @@
 )
 from lifx.exceptions import LifxTimeoutError
 from lifx.protocol import packets
+from numpy.ma import isin
```

Flake8 (7.3.0)

[error] 'numpy.ma.isin' imported but unused

(F401)